.circleci/config.yml

```diff
@@ -3,11 +3,11 @@ version: 2
 jobs:
   test:
     macos:
-      xcode: "13.0.0"
+      xcode: "13.3.0"
     environment:
-      - BUNDLE_JOBS: 3
-      - BUNDLE_RETRY: 3
-      - BUNDLE_PATH: vendor/bundle
+      BUNDLE_JOBS: 3
+      BUNDLE_RETRY: 3
+      BUNDLE_PATH: vendor/bundle
```

Comment:
Apparently this should have been a map but instead was an array of maps (I think)

> A map of environment variable names and values.

<img width="638" alt="Screenshot 2022-03-23 at 11 40 43" src="https://user-images.githubusercontent.com/482871/159690698-5bd86f3b-9d40-41e3-9454-02fa4475ed1d.png">